roles/module_openapi_cloud/templates/content_library_data.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+content_library_static_ds = {
```

Comment:
I'm wondering if templates is the correct directory to put this in, given this is a Python file containing a dictionary?